.github/workflows/docker.yml

```diff
@@ -0,0 +1,40 @@
+name: docker
+on:
+  push:
+    branches: [master, parachain]
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Nix Flakes doesn't work on shallow clones
+          fetch-depth: 0
+      - uses: cachix/install-nix-action@v13
+        with:
+          install_url: https://github.com/numtide/nix-flakes-installer/releases/download/nix-2.4pre20201221_9fab14a/install
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+      - uses: cachix/cachix-action@v10
+        with:
+          name: centrifuge-chain
+          authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
+      - run: nix build -L .#dockerContainer
+      - name: Load built image into Docker
+        run: docker load -i result
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+      - run: echo "BRANCH=$(echo $GITHUB_REF | cut -d'/' -f 3)" >> $GITHUB_ENV
+      - name: Tag image
+        if: env.BRANCH == 'master'
+        run: docker tag centrifugeio/centrifuge-chain:latest "centrifugeio/centrifuge-chain:$(date -u +%Y%m%d%H%M%S)-$(git rev-parse --short HEAD)"
+      - name: Tag image
+        if: env.BRANCH == 'parachain'
+        run: docker tag centrifugeio/centrifuge-chain:latest "centrifugeio/centrifuge-chain:parachain-$(date -u +%Y%m%d%H%M%S)-$(git rev-parse --short HEAD)"
+      - name: List images
+        run: docker images
+      - name: Push image to Docker Hub
```

Comment:
Does it mean that we will push `latest` even if it is a parachain build? If so, might want to avoid doing that, to minimize community confusion.